hibp.js

```diff
@@ -64,16 +62,35 @@ const HIBP = {
     return await this._throttledGot(url, reqOptions);
   },
 
+  // make list of all PNG logos
+  makeLogoListForEmails() {
+    const emailLogos = [];
+    const images = fs.readdirSync("./public/img/logos/");
+    images.forEach(file => {
+      if (file.endsWith("png")) {
```

Comment:
Can't imagine many cases where it'd cause an issue, or if you want to add slightly more footgun protection by using `.endsWith(".png")`.